src/SourceBuild/content/Directory.Build.props

```diff
@@ -110,7 +110,7 @@
     <IntermediateOutputPath Condition="'$(PlatformName)' != 'AnyCPU'">$(BaseIntermediateOutputPath)$(PlatformName)\$(Configuration)\</IntermediateOutputPath>
 
     <!-- TargetFrameworkDefaults.props -->
-    <NetCurrent>net9.0</NetCurrent>
+    <NetCurrent>net10.0</NetCurrent>
```

Comment:
We can't make this change yet. This will cause source build to build its bootstrapping tools targeting net10.0, using a 9.0 SDK (via global.json). So it's getting this error in the prep script:

`/vmr/.dotnet/sdk/9.0.100-preview.7.24407.12/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(166,5): error NETSDK1045: The current .NET SDK does not support targeting .NET 10.0.  Either target .NET 9.0 or lower, or use a version of the .NET SDK that supports .NET 10.0. Download the .NET SDK from https://aka.ms/dotnet/download [/vmr/eng/tools/BinaryToolKit/BinaryToolKit.csproj]`

This change couldn't be made until the global.json is updated to target an alpha version of 10.0.